### PR TITLE
#162638234 As a coach, I should be notified when a client completes their work plan. 

### PR DESCRIPTION
**Pull request by Quantum-35**
### What does this PR do?
 This PR add enables the coach to receive email notifications whenever a client completes their work plan.

### Description of Task to be completed?
- Notifies Coach whenever a client completes their Work Plan.
- Creates an Email template that is sent to the coach.

### How should this be manually tested?
On your terminal run;
- Check out into this branch `ft-notify-coach-on-workplan-completion-162638234`.
- run npm install.
- npm start.
- Login into the bot GUI and complete any task by typing  `B` to complete your work plan.

A coach will receive a descriptive email.

#### Email Template Picture .
screenshot 2019-01-30 at 12 51 22

What are the relevant pivotal tracker stories?
[#162638234](https://www.pivotaltracker.com/story/show/162638234)

Questions:
N/A

### CODE REVIEW

File: src/services/Email.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+      clientFirstName,
+      clientLastName,
+      coachFirstName,
+      coachEmail,
+      taskTitle,
+      url,
+      steps,
+      helpMessage
+    },
+  };
```
Comment (meshack-mbuvi) [inline src/services/Email.js]: I like what you have done on this task. Please consider adding a blank line after line 78

Comment (Quantum-35) [reply src/services/Email.js]: Noted.

File: src/Updater.js [hunk 2]
```diff
@@ -33,8 +34,18 @@ module.exports = class Updater {
       userAskedToStop,
       requestResolved,
       removeFollowup,
+      coachName,
+      coachEmail,
     } = this.variables;
-
+/* eslint-disable */
```
Comment (Quantum-35) [inline src/Updater.js]: Align line 40 with the rest of the code.

File: src/Updater.js [hunk 3]
```diff
@@ -59,6 +70,23 @@ module.exports = class Updater {
         return false;
       });
     }
+    
+  if(topic === TOPICS.ULTIMATE_DONE) {
```
Comment (Quantum-35) [inline src/Updater.js]: There should be a Space between if and the opening bracket.

File: src/services/Email.js [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        clientLastName,
+        coachEmail,
+        error: error.response.body
+      }
+    }
+    await sendgridClient.send(message);
+  }
+};
+
+const sendEmail = async (message) => {
```
Comment (Quantum-35) [inline src/services/Email.js]: Add `js docs` for all the functions in Email.js file.

File: src/services/Email.js [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+
```
Comment (Quantum-35) [inline src/services/Email.js]: Line 144 is unnecessary.

**Pull request merged**